src/components/Clasification.jsx

```diff
@@ -0,0 +1,106 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { fetchPokemon, fetchType } from '../redux/pokemon/pokemonSlice';
+import { filterType } from './Pokemon';
+import pokemonImg from '../assets/pokemon.png';
+
+const Clasification = () => {
+  let typesFiltered;
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [loadingType, setLoadingType] = useState(true);
+  const [loadingPokemon, setLoadingPokemon] = useState(true);
```

Comment:
- [ ] Finally update this react states to this. :+1: :100: 

```sh
  const [loadingType, setLoadingType] = useState(false);
  const [loadingPokemon, setLoadingPokemon] = useState(false);
```

## updating these 3 sections of your code will resolve this issue for you. :smile: :wave: :airplane:

Reply:
well resolved! ✔️